test/Microsoft.DotNet.Tools.Tests.Utilities/Commands/PackCommand.cs

```diff
@@ -23,6 +24,15 @@ private string OutputOption
                                            $"-o \"{_outputDirectory}\"";
             }
         }
+        private string BuildBasePathOption
+        {
+            get
+            {
+                return _buildBasePath == string.Empty ?
```

Comment:
`buildBasePath == string.Empty ` is common pattern for Command class options see https://github.com/dotnet/cli/pull/1535/files#diff-adee0a3160968bab49c8c4835c82cc86R61 https://github.com/dotnet/cli/pull/1535/files#diff-adee0a3160968bab49c8c4835c82cc86R51 and much more